posthog-core/src/index.ts

```diff
@@ -436,6 +438,10 @@ export abstract class PostHogCore {
           this.setKnownFeatureFlags(res.featureFlags)
         }
 
+        if (res.featureFlagPayloads) {
```

Comment:
need to bump decide version too! Also, don't publish new version until my changes go in too 😅

Reply:
Alternatively, merge this in after this: https://github.com/PostHog/posthog-js-lite/pull/53/files

Reply:
Yup waiting on that one